plugins/lookup/tss.py

```diff
@@ -270,13 +297,27 @@ def get_secret(self, term, fetch_file_attachments, file_download_path):
         else:
             return self._client.get_secret_json(secret_id)
 
+    def get_secret_ids_by_folderid(self, term):
```

Comment:
Remove the attempt to import the legacy thycotic SDK as this function doesn't exist there.

'from thycotic.secrets.server import SecretServer, SecretServerError'